source/nodejs/adaptivecards-designer/src/containers/host-container.ts

```diff
@@ -80,4 +91,40 @@ export abstract class HostContainer {
     get enableDeviceEmulation(): boolean {
         return false;
     }
+
+    // if various containers support different sizes in the future, we can override this method
+    static get supportedContainerSizes(): string[] {
```

Comment:
remove `static get` and move it to berlin container. don't need place holder for future. For example, the container size is only meaningful for Berlin.

Reply:
@licanhua I moved this method to BerlinContainer class. I kept it static though to avoid interating through `_hostContainers` to find the Berlin instance while creating the toolbar.